app/jobs/schedule_server_load_readings_job.rb

```diff
@@ -0,0 +1,22 @@
+class ScheduleServerLoadReadingsJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Server.where(status: :up).each do |server|
+      cpu = SshExecution.new(server).
+            execute(command: "uptime | awk '{print $10}' | tr -d '\n'").to_f
+      cpu = 100 if cpu > 100
```

Comment:
Will this basically only show "100" if you have 2 cores usage, at for example 150%?